libs/models/src/lib/file.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+import { IFile } from '@kol-amelamdim/types';
+
+const FilesSchema = new mongoose.Schema(
+  {
+    key: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    category: {
+      type: String,
+      required: true,
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    size: {
+      type: String,
+      required: true,
+    },
+    author: {
+      type: String,
+      required: true,
+    },
+    type: {
+      type: String,
+      required: true,
```

Comment:
we can add:

`validate(value) {
  // check if the file is one of the file types we support
}`

mongoose support this "validate" function inside models

Reply:
Resolved.